src/test/java/com/courtcanva/ccfranchise/utils/TestHelper.java

```diff
@@ -61,6 +74,33 @@ public static Franchisee createFranchiseeWithId() {
                 .build();
     }
 
+    public static Franchisee createMockFranchisee() {
```

Comment:
I guess the way to use the test helper is not a good idea since it will grow larger and messier when you are having  more tests